src/controller.cpp

```diff
@@ -127,6 +145,27 @@ void loop() {
     }
     sendState();
   }
+  checkTimer(); 
+}
+
+void startTimer(uint8_t channel) {
+	was_activated_at[channel-1] = clock()
+}
+
+void checkTimer() {
```

Comment:
The control logic seems to be right, so it should only be necesary getting around the time.h library.